info-web/src/test/resources/logback-dev.xml

```diff
@@ -5,7 +5,7 @@
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
             <charset>UTF-8</charset>
-            <pattern>%d [%-5level] [%thread] %logger{5} - [%X{consumerId}, %X{callId}, %X{userId}, %X{prosess}] - %m%n
+            <pattern>%d [%-5level] [%thread] %logger{5} - [%X{consumerId}, %X{callId}, %X{prosess}] - %m%n
```

Comment:
Kan vi bruke fødselsdato og maskere de 5 siste tallene ? Det gjøres andre steder

Reply:
Har du eksempel på oppsett av det?

Reply:
https://github.com/navikt/foreldrepengesoknad-api/blob/fc0a68bff824351f54743ed692c75a4eadc501d7/src/main/java/no/nav/foreldrepenger/selvbetjening/http/filters/HeadersToMDCFilterBean.java#L60

Forutsetter jo at dette er det eneste stedet id settes i MDC. 

Om det settes litt her og der må du mekke med pattern replace i logback
https://stackoverflow.com/questions/25052590/logback-replacing-10-digit-number-by-and-last-two-digit